pkg/controller/ingress-shim/sync_test.go

```diff
@@ -145,6 +151,7 @@ func TestSync(t *testing.T) {
 							Name: "issuer-name",
 							Kind: "ClusterIssuer",
 						},
+						Usages: defaultUsages,
```

Comment:
Can we include a case where the default usages are invalid?

Reply:
what do you mean exactly?